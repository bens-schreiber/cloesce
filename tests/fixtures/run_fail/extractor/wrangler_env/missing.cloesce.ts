// @ts-nocheck

@D1
export class Foo {
  @PrimaryKey
  id: number;
}

// no wrangler defined
