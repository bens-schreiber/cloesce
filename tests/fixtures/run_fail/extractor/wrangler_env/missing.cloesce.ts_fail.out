==== CLOESCE ERROR ====
Error [MissingWranglerEnv]: Missing a wrangler environment definition in the project
Phase: TypeScript IDL Extraction
Suggested fix: Add a @WranglerEnv class in your project.
